demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -447,3 +453,79 @@ def test_DuplicateContentIdValidatorAllFiles_is_invalid(prepared_graph_repo: Rep
         DuplicateContentIdValidatorAllFiles().obtain_invalid_content_items([])
     )
     assert len(validation_results) == 4
+
+
+@pytest.fixture
+def repo_for_test(graph_repo):
+    # A repository with 3 packs:
```

Comment:
let's add more info - what do the 3 packs include?

Reply:
Thats not my work, obviously...:)